index.php

```diff
@@ -379,3 +408,17 @@ function the_gutenberg_project() {
 	</div>
 	<?php
 }
+
+/**
+ * Server side block renderer for core/oembed.
+ *
+ * @param array $content Block data
+ *
+ * @returns string Rendered block content, using WP_oEmbed.
+ */
+function gutenberg_block_core_oembed( $content ) {
+	$oembed = new WP_oEmbed();
+	return $oembed->get_html( $content['url'] );
+}
+
+register_block( 'core/oembed', 'gutenberg_block_core_oembed' );
```

Comment:
Aside: This index.php is getting bigger and bigger, we may want to split it soon.